rpc/js/.npmignore

```diff
@@ -1,4 +1,3 @@
 etc
 node_modules
-tsconfig.json
-webpack.config.js
+tsconfig.json
```

Comment:
Rather than maintain a `.npmignore` file, it's usually easier to specify an allowlist via the `files` array in package.json